### PR TITLE
Create wrapper for `makemigrations`

### PR DESCRIPTION
**Pull request by saponifi3d**
# Description
I've been creating a number of new data models recently, but it's been fairly tricky to understand all the places and commands to run.

As a quick fix to the issue, proposing this bin script file that wraps all the automation that needs to run.

I've confirmed this script works as expected:
- when there isn't a migration file
- if `-h` or `--help` is passed in
- and if a migration is successfully configured.

I also was able to reproduce some of the failure scenarios i've seen on CI, locally with the 3rd command (things missing from backups.py, related to the relocation scope).

### CODE REVIEW

**Comment by saponifi3d** [comment]
Lemme know what y'all think - this command would probably have saved at least a few days of dev on my latest project.

I think ideally, we remove the need to execute things outside of `sentry makemigrations` but this can be a bandaid while those discussions are happening.

File: src/sentry/management/commands/makemigrations.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def handle(self, *app_labels, **options):
                 "Please name your migrations using `-n <migration_name>`. "
                 "For example, `-n backfill_my_new_table`"
             )
-            return
+            sys.exit(1)
```
Comment (saponifi3d) [inline src/sentry/management/commands/makemigrations.py:51]: added this becasue the `makemigrations` was returning successfully for the error state. This change will allow the `make-migration` bin script to correctly short circuit.

**Comment by asottile-sentry** [comment]
I would rather we just nuke the snapshots -- they aren't providing value as far as I can tell

**Comment by saponifi3d** [comment]
> I would rather we just nuke the snapshots -- they aren't providing value as far as I can tell

@asottile-sentry agreed, but that feels like it could take a little bit for us to get everyone to align to that as well. I was thinking of this as a stop gap measure to save a _lot_ of additional effort when creating models today (multiple days of just tedious maintenance).

**Comment by asottile-sentry** [comment]
> > I would rather we just nuke the snapshots -- they aren't providing value as far as I can tell
> 
> @asottile-sentry agreed, but that feels like it could take a little bit for us to get everyone to align to that as well. I was thinking of this as a stop gap measure to save a _lot_ of additional effort when creating models today (multiple days of just tedious maintenance).

I mean, I don't want to align on _this_ either :P -- easy enough to just make a shell alias in the meantime

**Comment by saponifi3d** [comment]
> I mean, I don't want to align on this either :P -- easy enough to just make a shell alias in the meantime

👍 yep, i already have one. This seems like this is a larger scope issue though, and there isn't anything happening to address the problem.

These commands are not documented, and the only way you can see them is for CI failures. This simple script can at least unblock people when they're making new models (rather than spending days trying to find all the undocumented commands).

**Pull request closed without merging**